pkg/users/accounts-list.js

```diff
@@ -382,6 +382,7 @@ const AccountsList = ({ accounts, current_user, groups, min_uid, max_uid, shells
         { title: _("ID"), sortable: true },
         { title: _("Last active"), sortable: true },
         { title: _("Group") },
+        { title: <span className="pf-v5-u-screen-reader">{_("Actions")}</span> },
```

Comment:
Completely incidentally I did something similar in https://github.com/cockpit-project/cockpit-podman/pull/1977 -  let's give it a `screenReaderText` property as well here? [commit](https://github.com/cockpit-project/cockpit-podman/pull/1977/commits/56441f0c1cb2c24d7aa9b67da7ed6077ef15f35f)

Reply:
Oh good spot, didn't see this before. That is a lot better tbh!

Reply:
Applied commit style and works fine. Gonna wait and see how tests look